CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+
+Fixes and changes which address the actual behaviour or meaning of a specification *after they have been released*, such as [#204](https://github.com/ircv3/ircv3-specifications/pull/204) generally require 'errata updates'. Examples of errata updates can be found in various specifications, but they basically leave a log as to how a specification has been changed after it was released. Before submitting a fix of this type, you should talk with other members on the [#ircv3](http://ircv3.net/contact.html) channel and ensure there is support for your change.
+
+## Ideas
+
+Feature proposals contributed to the IRCv3-Specifications repository require a concrete specification and at least one reference or existing implementation.
+
+If you have an idea or a protocol proposal doesn't have these things or is not yet at this stage, you should make an issue on the [IRCv3/ideas](https://github.com/ircv3/ideas) repository instead. This repository is intended for working group members, as well as developers and users to be able to suggest ideas that are not yet full realised or without a reference implementation.
+
+Contributing to the [IRCv3/ideas](https://github.com/ircv3/ideas) repository does not have any stakeholder requirements. It was created so that contributors may decide to only review ideas that have already been specified and have support from other stakeholders of the IRCv3 WG, or review those that aren't yet at that stage.
+
+## Feature Proposals
+
+A new feature proposal includes things like [`account-tag`](http://ircv3.net/specs/extensions/account-tag-3.2.html), [`cap-notify`](http://ircv3.net/specs/extensions/cap-notify-3.2.html) and [`echo-message`](http://ircv3.net/specs/extensions/echo-message-3.2.html).
+
+Contributing a proposal to the IRCv3-Specifications repo requires a few things. These include:
+
+### Stakeholder Support
+
+Specifications submitted to this repository should be proposed or sponsored by a stakeholder of the IRCv3 Working Group.
+
+### Concrete Specification
+
+Your specification *must* describe all behaviour that comes with your proposal. You cannot simply say *"Yeah, and this part we'll work out later, it should be fine."*.
+
+The protocol syntax specified in your proposal may, of course, be changed after it is submitted. However, it should *not* be submitted with the attitude of *"Well, I didn't think about this too hard and I don't think it's great, what do other people think?"*. Proposals of that type should go through the [IRCv3/ideas](https://github.com/ircv3/ideas) process before being submitted to this repository.
+
+### Vendor Tag
+
+Any capabilities that your proposal introduces *must* be vendor-tagged. The vendor tag will be removed once it is made into a proper specification through the IRCv3 release process.
+
+For instance, if my spec introduces a new capability which will be named `cool-tag`, in the spec and in the reference implementation it should be specified as something like `example.com/cool-tag`.
```

Comment:
Not necessary... There was a sad story of `znc.in/server-time` which was rewritten and became `znc.in/server-time-iso`, which became `server-time` as is. If spec is changed, it can't preserve the same name after `/`